<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSettingsWindow</class>
 <widget class="QDialog" name="FilterSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_src_host">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Source host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dst_host">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Destination host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_src_port">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Source port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dst_port">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Destination port</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_protocol">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>All</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UDP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_temp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_src_host">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Not</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dst_host">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Not</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_src_port">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Not</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dst_port">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Not</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterSettingsWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterSettingsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
